src/common/decorators/validators/is-not-empty-object-and-all-false.decorator.ts

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { applyDecorators } from '@nestjs/common';
+import { Type } from 'class-transformer';
+import {
+  IsNotEmptyObject,
+  ValidationArguments,
+  ValidationOptions,
+  registerDecorator,
+} from 'class-validator';
+
+export function IsNotEmptyObjectAndAllFalse(
+  options?: { nullable: boolean },
+  validationOptions?: ValidationOptions,
+) {
+  return function (object: any, propertyName: string) {
+    IsNotEmptyObject(options, validationOptions)(object, propertyName);
+    registerDecorator({
+      name: 'isNotEmptyObjectAndAllFalse',
+      target: object.constructor,
+      propertyName: propertyName,
+      constraints: [],
+      options: validationOptions,
+      validator: {
+        validate(value: any, args: ValidationArguments) {
+          for (const key in value) {
+            if (value[key]) {
+              return true;
+            }
+          }
+
+          return false;
+        },
```

Comment:
```suggestion
return Object.values(value).some(())
```
이런식으로 처리 가능할듯요
메서드 내부는 직접 채워서